pkg/v1/service/books/books.go

```diff
@@ -0,0 +1,10 @@
+package books
+
+import (
+    "context"
+)
+
+// GetBook returns the details of a given book ID. 
+func (s *bookSvc) GetBook(ctx context.Context, id string) (Book, error) {
+    return Book{Name: "Perdido Street Station"}, nil
```

Comment:
Did you have thoughts on a data store? SQLite for example?

Reply:
Can we deploy with SQLite? I was thinking MySQL or Postgresql.

Reply:
For two people? Yeah SQLite works.